packages/bff/src/routers/catalogRouter.ts

```diff
@@ -152,10 +151,46 @@ const catalogRouter = (
       "/eservices/:eServiceId/descriptors/:descriptorId/documents/:documentId/update",
       async (_req, res) => res.status(501).send()
     )
-    .delete("/eservices/:eServiceId", async (_req, res) =>
-      res.status(501).send()
-    )
-    .put("/eservices/:eServiceId", async (_req, res) => res.status(501).send())
+    .post("/eservices", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+      try {
+        const createdResource = await catalogService.createEService(
+          req.body,
+          ctx
+        );
+        return res.status(200).send(createdResource);
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
```

Comment:
We should add the logs to `makeApiProblem` as you did in #862 using the same messages used in Scala (here and in the other calls). Anyways we will do a check at the end of the BFF migration probably, to make sure we did it everywhere...

Reply:
[6b0b0a0](https://github.com/pagopa/interop-be-monorepo/pull/885/commits/6b0b0a0004866c8cf22d968ac395296d729e3f2b)